README.md

```diff
@@ -18,7 +18,8 @@ Next to `numpy`, `scipy`, and `matplotlib` that are included in the standard
 anaconda distribution, you will need to install the following dependencies
 to be able to use `mne_bids`:
 
-    $ pip install pandas mne
+    $ pip install pandas
+    $ pip install -U https://api.github.com/repos/mne-tools/mne-python/zipball/master
```

Comment:
maybe we can note the mne-python current version as the minimal in the text above

Reply:
not sure what you mean?

Reply:
I guess we don't specify anywhere minimal version number. like, we should have a section that says mne >= 0.17.X

Reply:
That's probably because we don't know them. I know only for `MNE` we need >=0.17.x. For the rest I would say the "latest" :-)